options/locale/locale_en-US.ini

```diff
@@ -517,7 +517,7 @@ appearance = Appearance
 password = Password
 security = Security
 avatar = Avatar
-ssh_gpg_keys = SSH / GPG Keys
+keys = Keys
```

Comment:
IMO: I think `Keys` is too vague to be a navbar item.

Reply:
I suspect that we might need to have a `GPG Keys` menu option if SSH is absolutely disabled.

Reply:
Menu item should not contain technology name (SSH, GPG, etc.) because disabled items should be removed from page (clean UI should not contain stuff user don't use). It should also be short and meaningfull. "Keys" are fine IHMO and not more vague than other existing items there. Propose better name if you can't accept simple "Keys".

Reply:
If you check GitHub's UI and Gitlab's UI you'll find that the technology name is included in the tab name.

We have to include the technology type to disambiguate the type of keys.

There are authentication keys - eg. Yubikeys which are not configured on this tab. If you call it Keys then this will become confusing.

Reply:
> If you check GitHub's UI and Gitlab's UI you'll find that the technology name is included in the tab name.

Gitea may have cleaner UI :)

> There are authentication keys - eg. Yubikeys which are not configured on this tab. 

SSH keys are also authentication keys.

Propose better name if you can't accept simple "Keys".

Reply:
Sorry didn't noticed `GPG Keys` proposal - will extend mod with this. Still consider merging all this auth/sign/token stuff in one page in user settings in the future.

Reply:
Done.